src/services/platform.service.js

```diff
@@ -2,10 +2,11 @@ import * as childProcess from 'child_process';
 import * as os from 'os';
 import * as path from 'path';
 import { remote } from 'electron';
+
 import { PACKAGE_MANAGER } from '../config/app';
 
-// Returns true if the OS is Windows
 export const isWin = /^win/.test(os.platform());
+export const isMac = /darwin/.test(os.platform());
```

Comment:
the original code used __DARWIN__, where was that coming from? Was that also something that tested os.platform()?

Reply:
Originally, it was this:

```
const {process} = require('electron').remote;

const __DARWIN__ = process.platform === 'darwin';	
```

So, essentially I shifted it from using Electron's `remote.process.platform` to using Node's `os.platform()`. Do you think this is significantly different? On my iMac, the two return identical values.

The reason for the switch was twofold:
- Consistency with `isWin`
- The tests were failing, because `remote.process.platform` is undefined when running Jest tests. I think I could've mocked this out, but I didn't take the time to figure it out.

Reply:
Also, `remote.process.platform` has to do a sync IPC call, which is slower than doing the same test within one process.

Reply:
I took the `__DARWIN__` pattern from Github's Desktop app. at @mathieudutour's suggestion. It was in the midst of the Windows crunch, so I didn't pay it much mind.

@j-f1 is right, this should be more efficient and also fits our code style better. +1